### PR TITLE
Add Chinese document for how to use Hango UI

### PR DESCRIPTION
**Pull request by nevermosby**
**Description**

Add Chinese document for how to use Hango UI

**Affects**

To help us figure out who should review this PR, please put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Test and Release
- [ ] User Experience

### CODE REVIEW

File: example/expose_api_with_ui.zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+hango-ui          NodePort    10.100.82.153    <none>        8789:30829/TCP                                    23h
+istio-e2e-app     ClusterIP   10.100.15.26     <none>        80/TCP,8080/TCP,90/TCP,9090/TCP,70/TCP,7070/TCP   23h
+istiod            ClusterIP   10.109.241.243   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP             23h
+plugin            ClusterIP   10.96.91.4       <none>        80/TCP                                            23h
+slime-metrics     ClusterIP   10.102.87.122    <none>        8383/TCP,8686/TCP                                 23h
+```
+通过主页README说明如下：
+- `gateway-proxy`
+
+其中`hango-ui`是我们的目标服务，它是NodePort类型，可以通过主机IP地址+端口访问，可以在浏览器中输入如下地址，即可访问`Hang UI`：
```
Comment (ethanhanjiahao) [inline example/expose_api_with_ui.zh_CN.md:23]: Hang UI  -> Hango UI

File: example/expose_api_with_ui.zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+hango-api-plane   ClusterIP   10.111.150.106   <none>        10880/TCP                                         23h
+hango-portal      NodePort    10.107.179.97    <none>        80:30621/TCP                                      23h
+hango-ui          NodePort    10.100.82.153    <none>        8789:30829/TCP                                    23h
+istio-e2e-app     ClusterIP   10.100.15.26     <none>        80/TCP,8080/TCP,90/TCP,9090/TCP,70/TCP,7070/TCP   23h
+istiod            ClusterIP   10.109.241.243   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP             23h
+plugin            ClusterIP   10.96.91.4       <none>        80/TCP                                            23h
+slime-metrics     ClusterIP   10.102.87.122    <none>        8383/TCP,8686/TCP                                 23h
+```
+通过主页README说明如下：
+- `gateway-proxy`
```
Comment (ethanhanjiahao) [inline example/expose_api_with_ui.zh_CN.md:21]: ?

File: example/expose_api_with_ui.zh_CN.md (hunk 1 above)
Comment (nevermosby) [reply example/expose_api_with_ui.zh_CN.md]: Fixed, thx for remind

File: example/expose_api_with_ui.zh_CN.md (hunk 2 above)
Comment (nevermosby) [reply example/expose_api_with_ui.zh_CN.md]: Just deleted.

**Comment by nevermosby** [comment]
Just updated, pls help review again @ethanhanjiahao

**Comment by ethanhanjiahao** [comment]
LGTM

**Pull request merged**